r/R/dplyr-funcs.R

```diff
@@ -58,15 +58,27 @@ NULL
 #' @keywords internal
 #'
 register_binding <- function(fun_name, fun, registry = nse_funcs) {
-  name <- gsub("^.*?::", "", fun_name)
-  namespace <- gsub("::.*$", "", fun_name)
+  qualified_name <- fun_name
```

Comment:
Why not just keep it as `fun_name`? It's kinda odd when the first line of a function just renames the function argument.

Reply:
Suggestion to change the name was part of an earlier review: https://github.com/apache/arrow/pull/13160#discussion_r914801827